app/views/shared/_nav.haml

```diff
@@ -6,3 +6,6 @@
   %section
     %h2 Checkins By Date
     %ul= render partial: 'shared/dates', locals: { dates: @dates }
+  %section
+    %h2 Checkins By Cohort
```

Comment:
Even if we keep the term 'cohort' in the codebase, this line should probably say 'Checkins by Class' so as to minimize confusion for users.